### PR TITLE
feat: Add new variants to Global Notification [KDS-49]

### PR DESCRIPTION
**Pull request by christian314159**
## Objective
Add missing `persistent` variants to GlobalNotification component

So far all global notifications were always _dismissible_, I've added `persistent` as a new prop, which by default is `false` so that there are no changes required to current usages.

*Update (BREAKING CHANGE)*
The `affirmative` variant is now called `positive` 

I also cleaned up the related story:

image

### CODE REVIEW

**Review by NatalieLoyzaga — APPROVED** [review]
Storybook looks great!

**Comment by jmitsios** [comment]
@christian314159 

**Changes to make:**
- All label instances of "**Affirmative**" should be replaced with "**Positive**"
- We'd like to have the message descriptions reflect the ones written inside the Heart UI Kit:
**Positive**: _Emails will be sent notifying coaches and inviting reviewers to give their feedback. View all_
**Informative**: _[Survey name]’s status has been changed to ‘Archived’.  View all_
**Negative**: _Something went wrong while validating and analyzing user data. View all_ 
**Cautionary**: _The syncing process can take some time to complete. Keep this window open until complete. View all_

**Questions:**
- Does the Kaizen team have a defined structure for what a story should look like?  There's been inconsistencies across each story;  _example: Do we want just one story (sticker sheet) for all variants of a component? Should there be consumer controls?_
- Will we be moving towards a story that allows the consumer to enable/disable the persistent boolean?

**Food for thought for Page Layout work in 2022:**
- Standardise the positioning of where the **Global Notifications** sit - to ideally place them _above_ the navigation menu. Let's discuss.

**Comment by ckychris** [comment]
This is a test comment

**Comment by ckychris** [comment]
This is yet another test comment for verifying if chromatic works.

**Comment by ckychris** [comment]
What does run chromatic do? let's find out.

File: packages/notification/src/components/GenericNotification.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import successIcon from "@kaizen/component-library/icons/success.icon.svg"
 import styles from "./GenericNotification.module.scss"
 
 export type NotificationType =
-  | "affirmative"
+  | "positive"
```
Comment (dougmacknz) [inline packages/notification/src/components/GenericNotification.tsx:13]: Just making sure you're planning to release this as a breaking change?

Comment (christian314159) [reply packages/notification/src/components/GenericNotification.tsx:13]: @dougmacknz Yes, definitely. I was actually just looking up if it suffices to do it in the final commit/merge message or whether it has to be in the PR title.

Comment (dougmacknz) [reply packages/notification/src/components/GenericNotification.tsx:13]: You're right, it just needs to be in the final commit/merge message.

**Comment by jmitsios** [comment]
@christian314159 All changes look good to me, no feedback!

**Review by mcwinter07 — COMMENTED** [review]
All looks positive to me 👍  I affirm that this looks good to merge

**Review by mcwinter07 — COMMENTED** [review]
Doubling up on the comments since I forgot to actually check the view boxes 😆  But all looks good to merge 👍

**Review by mcwinter07 — APPROVED** [review]
Approved for merge 👍

**Pull request merged**